nestjs-counter/apps/server/src/app.module.ts

```diff
@@ -0,0 +1,8 @@
+import { Module } from '@nestjs/common';
```

Comment:
Can you fix these lint warnings? Looks like we need to add decorator support to eslint somehow..

Reply:
Where are the lint warnings? It looks like lint succeeds in CI...

Reply:
<img width="694" alt="image" src="https://user-images.githubusercontent.com/52304/195471897-ae352ddc-0aea-484e-8054-d512b344707a.png">